### PR TITLE
Fixed Linter Errors so test runs.

### PR DESCRIPTION
**Pull request by movshov**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Fixed errors preventing linter from working. 

* **What is the current behavior?** (You can also link to an open issue here)

Linter no work

* **What is the new behavior (if this is a feature change)?**

Linter work

* **Other information**:
JESUS CHRIST LOTS OF WORK

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -380,6 +381,8 @@
           "requires": {
             "lru-cache": "^4.0.1",
             "which": "^1.2.9"
+=======
+>>>>>>> Fixed Linter Errors so not it runs.
```
Comment (Liriaene) [inline package-lock.json:385]: I'm not sure if I am reading this correctly, but it looks like a bunch of merge conflicts got added to the branch?
These lines will probably break the file.

File: src/app/summary/summary.component.ts [hunk 2]
```diff
@@ -30,10 +31,16 @@ export class SummaryComponent implements OnInit {
     console.log('landing page');
   }
 
+=======
+>>>>>>> Fixed Linter Errors so not it runs.
```
Comment (Liriaene) [inline src/app/summary/summary.component.ts:35]: Some more of those merge conflict messages being added I think

**Review by Liriaene — CHANGES_REQUESTED** [review]
I think something happened during merging and now a bunch of gunk got added.

File: package-lock.json (hunk 1 above)
Comment (ElijahRW) [reply package-lock.json:385]: Ironically, it doesn't as long as you rerun `npm install` but it should be fixed regardless

File: src/app/summary/summary.component.ts (hunk 2 above)
Comment (gstone271) [reply src/app/summary/summary.component.ts:35]: Yeah this needs to be gone, I'm surprised it compiles.

**Pull request closed without merging**